tests/fast/AtomicBackupToDBCorrectness.toml

```
@@ -1,5 +1,7 @@
 [configuration]
 extraDatabaseMode = 'LocalOrSingle'
+# DR is not currently supported in required tenant mode
+	tenantModes = ['disabled', 'optional']
```

There seems to be some extra whitespace on this and similar test specs:

```suggestion
tenantModes = ['disabled', 'optional']
```